tests/foreman/ui_airgun/test_ldap_authentication.py

```diff
@@ -763,3 +770,88 @@ def test_positive_add_katello_role_with_org(
     ak = entities.ActivationKey(organization=module_org).search(
         query={'search': 'name={}'.format(ak_name)})[0].read()
     assert ak.organization.id == module_org.id
+
+
+@skip_if_not_set('ldap')
+@tier2
+@upgrade
+def test_positive_create_user_in_ldap_mode(session, auth_source):
+    """Create User in ldap mode
+
+    :id: 0668b2ca-831e-4568-94fb-80e45dd7d001
+
+    :expectedresults: User is created without specifying the password
+
+    :CaseLevel: Integration
+    """
+    auth_source_name = 'LDAP-' + auth_source.name
+    name = gen_string('alpha')
+    with session:
+        session.user.create({
+            'user.login': name,
+            'user.auth': auth_source_name,
+        })
+        assert session.user.search(name)[0]['Username'] == name
+        user_values = session.user.read(name)
+        assert user_values['user']['auth'] == auth_source_name
```

Comment:
shouldn't we perform some sort of cleanup in the end not to spam ldap server with pointless users?

Reply:
it will create user only on local Satellite server, just use ldap authentication for its work